src/components/nowPlayingBar/nowPlayingBar.js

```diff
@@ -660,7 +660,7 @@ import { appRouter } from '../appRouter';
         console.debug('nowplaying event: ' + event.type);
         const player = this;
 
-        if (!state.NowPlayingItem || layoutManager.tv || !state.IsFullscreen) {
+        if (!state.NowPlayingItem || layoutManager.tv) {
```

Comment:
This change reverts https://github.com/jellyfin/jellyfin-web/pull/2028. Should we handle that differently somehow?

Reply:
Probably, but the main issue is that it's using the "music" player to play theme songs, which is of course attached to all the audio playing stuff in Jellyfin (playlist, controls, etc).

So short of reworking theme songs, I don't see a good way to handle this :/

Reply:
`state.IsFullscreen` is only set when playback starts. Probably should be updated in this function https://github.com/jellyfin/jellyfin-web/blob/e56414ebe8876f6ba23a667ddbdc6b0dc8264678/src/components/playback/playbackmanager.js#L1932

Quick fix.
```suggestion
        if (!state.NowPlayingItem || layoutManager.tv || state.IsFullscreen === false) {
```

Reply:
Nice! It works.